### ISSUE TITLE
Discrepancy between results

### ISSUE THREAD

**Issue by Tamme** [giandbt/synboost#9]
Hi and thank you for a great algorithm!

Why is there discrepancy between results in tables 1 and 2? 
Eg. FS L&F:
AP 43 vs 55
FPR95 15 vs 40

All the best in future developments!

**Comment by giandbt** [comment]
Hi Tamme, 
The difference is that Table 1 is the test sets for Fishyscapes while Table 2 is just the validation sets. We were only allowed one submission for Fishyscapes so we sent our best model. We couldn't do that for the ablation study.  

The drop in performance is related to the difference between the two sets and also some overfitting the model had with the validation set. The test sets are significantly larger than the validation sets and cover more objects in different scenarios.

**Comment by Shanmukh-C** [comment]
Hello,
I could not find seperate  dataset for validation and testing. As even the validation dataset is same as test_dataset3 (fs lost and found). And as far as I have understood with the datasets provided in the repo we can reproduce only table2.

I would be grateful if you provide the dataset which you used to find metric in table1.
Thank You
